easydict/__init__.py

```diff
@@ -112,6 +132,17 @@ def __setattr__(self, name, value):
 
     __setitem__ = __setattr__
 
+    def update(self, E=None, **F):
```

Comment:
we usually avoid capital letters in arguments but since it's like this in the upstream repo I can live with it

https://github.com/python/cpython/blob/fcd5e84a515e19409840c570730f0728e9fcfc83/Objects/dictobject.c#L3050-L3054

Reply:
I thought it's better to stick with lowercase (since `pop` doesn't use capital letter, urrrr.....).
Maybe `update(self, e=None, **f)` looks better.

Reply:
Also, we don't call super method of `update`